jobs/suite/default.env

```diff
@@ -1,2 +1 @@
-GINKGO_TEST_ARGS=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]
 GINKGO_PARALLEL=y
```

Comment:
ack - maybe just move the env to their original env files... those `suite/*.env` files are not loved anymore 🔥